packages/daemon/src/daemon-node-powers.js

```diff
@@ -279,111 +375,208 @@ export const makePowers = ({
     }
   };
 
-  /**
-   * @param {string} path
-   */
-  const makeFileReader = path => {
-    const nodeReadStream = fs.createReadStream(path);
-    return makeNodeReader(nodeReadStream);
+  const makeHashedContentReadeableBlob = (statePath, sha512) => {
```

Comment:
naming is hard

Reply:
It is. How about `makeReadableBlobForContentHash`?